cluster/config-defaults.yaml

```diff
@@ -70,6 +70,14 @@ skipper_ingress_min_replicas: "2"
 {{end}}
 skipper_ingress_cpu: "1000m"
 skipper_ingress_memory: "1500Mi"
+
+# Enables deployment of canary version
+skipper_ingress_canary_enabled: "false"
```

Comment:
Let's enable it for all non production clusters!

{{if eq .Cluster.Environment "production"}}
skipper_ingress_canary_enabled: "false"
{{else}}
skipper_ingress_canary_enabled: "true"
{{end}}

Reply:
Lets do this via cluster registry, then we can put it under condition